README.md

```diff
@@ -225,8 +198,9 @@ transactionRequest.run(new ResultCallback<TransactionId>() {
 ```
 
 #### Memo
+
 Arbitrary data can be added to a transfer operation using the memo parameter,
-the memo is a `String` of up to 28 characters.
+the memo can contain a utf-8 string up to 28 bytes in length. A typical usage is to include an order number that a service can use to verify payment.
```

Comment:
Mention it should be used to identify the transaction

Reply:
it's not precisely to identify the transaction, you have transaction ID for that (hash) but to verify the payment like mentioned. anyhow, a clear use case can be added that will explain this point, to our future use cases section...